RELEASE_NOTES.md

```diff
@@ -9,20 +9,28 @@
 [//]: # ( - Configurations public constructor are deprecated, please use each Configuration's builder to make a Configuration object)
 
 ## New
-- Address Lookup functionality for Card Component. You can enable this feature by setting your address configuration to lookup while building your card configuration as follows
-```kotlin
-CheckoutConfiguration(
-    shopperLocale = shopperLocale,
-    environment = environment,
-    clientKey = clientKey,
-    amount = amount,
-    analyticsConfiguration = createAnalyticsConfiguration(),
-) {
-    card {
-        setAddressConfiguration(AddressConfiguration.Lookup())
+- Address Lookup functionality for Card Component. 
+  - You can enable this feature by setting your address configuration to lookup while building your card configuration as follows:
+    ```kotlin
+    CheckoutConfiguration(
```

Comment:
How about moving the release note where we mention that we added `CheckoutConfiguration` to the top? Right now if you read the release notes you won't understand where did  `CheckoutConfiguration` come from until you reach it later